lib/train/file/local/unix.rb

```diff
@@ -0,0 +1,77 @@
+# encoding: utf-8
+
+require 'shellwords'
+require 'train/extras/stat'
+
+module Train
+  class File
+    class Local
+      class Unix < Train::File::Local
+        def sanitize_filename(path)
+          @spath = Shellwords.escape(path) || @path
+        end
+
+        def stat
+          return @stat if defined? @stat
```

Comment:
Slight nitpick, but we should be consistent with wrapping method arguments with parentheses...

```ruby
return @stat if defined?(@stat)
```